pkg/common/config/types.go

```diff
@@ -63,8 +63,11 @@ type Config struct {
 		Zone   string `gcfg:"zone"`
 		Region string `gcfg:"region"`
 	}
-	// FeatureStatesConfig is the details about feature states configmap
+	// FeatureStatesConfig contains feature states configmap info specific to supervisor cluster
 	FeatureStatesConfig FeatureStatesConfigInfo
+
+	// InternalFeatureStatesConfig contains feature states configmap info specific to pvCSI and vanilla drivers
```

Comment:
Can we add a comment that this internal FSS should not be edited by the customer and is used only for dev and test purposes?

Reply:
Sure, will add a comment

Reply:
Can we track a doc task for clarifying this in the CSI doc? We do not want customers to play around with this config map. So, tracking this as a doc bug will help us remember.